2-mandatory/4-stocks.js

```diff
@@ -60,11 +77,15 @@ function getPriceChanges(closingPricesForAllStocks) {
         - Returns an array of strings describing what the highest price was for each stock.
             For example, the first element of the array should be: "The highest price of AAPL in the last 5 days was 180.33"
             The test will check for this exact string.
-    The stock ticker should be capitalised.
+    The stock ticker should be capitalized.
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+    highestPriceLast5Days = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+        highestPriceLast5Days.push(`The highest price of ${stocks[i].toUpperCase()} in the last 5 days was ${Math.max(...closingPricesForAllStocks[i]).toFixed(2)}`);
+    }
```

Comment:
I have seen it on google using Math.max() it really helps to make easier our work, anyway I also used another way to solve it.